### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by OscarFMdev**
# PR description:
In this branch I:
- Added the project low fidelity wireframe. 📑 
- Added a JavaScript function to simulate user data input. 💻 

### Thanks for reviewing my code! 🙌

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,9 +3,36 @@
...
+      <section class="leaders">
+        <h2 class="leaders-heading">Recent scores <button>Refresh</button></h2>
+        <ul class="leaders-container">
+          <!-- <li class="leader-1">Name: 100</li>
+          <li class="leader-0">Name: 20</li>
+          <li class="leader-1">Name: 50</li>
+          <li class="leader-0">Name: 78</li>
+          <li class="leader-1">Name: 125</li>
+          <li class="leader-0">Name: 77</li>
+          <li class="leader-1">Name: 42</li> -->
```
Comment (ichala) [inline src/index.html]: - [ ] to make your project looks clean and professional kindly get rid of any commented code in your project 🙏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @OscarFMdev , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

-  Gitflow followed✔️ 
- No linters Errors ✔️
- Webpack Installed ✔️


 ## Required Changes ♻️


- [ ] 🛑 Kindly update the setup&install section in your readme file to info the user on how to install your project and run it , example : run `npm install` , `npm start` etc ...
- [ ] 🛑 **_Check The Comments below_**

## Optional suggestions

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShoiraTa — COMMENTED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @OscarFMdev , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

### Highlights:
 ✔️ Project is setup correctly
 ✔️ All linters are passing
✔️ Correct gitflow
 ✔️ Professional documentation

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

image

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@shoirata] in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by OscarFMdev** [comment]
Thank you @ichala and @ShoiraTa for reviewing my code! 🙌

**Pull request merged**